cluster/cluster_members.go

```diff
@@ -270,3 +270,56 @@ func GetClusterMemberAPIExtensions(ctx context.Context, tx *sql.Tx) ([]extension
 
 	return results, nil
 }
+
+// GetUpgradingClusterMembers returns the list of all cluster members during an upgrade, as well as a map of members who we consider to be in a waiting state.
+// This function can be used immediately after dqlite is ready, before we have loaded any prepared statements.
```

Comment:
Would it be worth adding a comment saying a member is still marked to be in a waiting state if the schemas match but not yet the API extensions (as the latter takes precedence in the returned map)?